futures-util/src/future/future/mod.rs

```diff
@@ -112,6 +111,36 @@ mod shared;
 #[cfg(any(feature = "std", all(feature = "alloc", feature = "spin")))]
 pub use self::shared::{Shared, WeakShared};
 
+/// Pins a value on the stack.
+///
+/// **Note:** Since Rust 1.68, this macro is soft-deprecated in favor of
+/// [`pin!`](https://doc.rust-lang.org/std/pin/macro.pin.html) macro
+/// in the standard library.
```

Comment:
Why dropped [this line](https://github.com/rust-lang/pin-utils/blob/c635c0ca87cd9f9173152f367ef1d84df45943ab/src/stack_pin.rs#L3) from the original doc?

Reply:
Should be by accident. Will add it back.